test/cpp/tensorexpr/test_conv.cpp

```diff
@@ -10,6 +12,44 @@ namespace jit {
 namespace te = torch::jit::tensorexpr;
 namespace F = torch::nn::functional;
 
+// Generate test data with few bits of precision, to minimize error
+// accumulation from floating-point reordering.
+static at::Tensor genTestData(c10::IntArrayRef args) {
+  return at::trunc(at::randn(args) * 256.0f) / 256.0f;
+}
+
+TEST(Conv, DepthwiseConv2D) {
+  te::KernelScope kernel_scope;
+  constexpr int N = 1, C = 72, H = 56, W = 56;
+  constexpr int K = 72, R = 3, S = 3;
+  constexpr int kPad = 1, kStride = 2, kGroups = C;
+  constexpr int CperG = C / kGroups;
+
+  te::Placeholder input("input", te::kFloat, {N, C, H, W});
+  te::Placeholder weight("weight", te::kFloat, {K, CperG, R, S});
+  te::Placeholder bias("bias", te::kFloat, {K});
+  te::Tensor* output = te::conv2d_depthwise(
+      input.handle(), weight.handle(), bias.handle(), kStride, kPad, kGroups);
+
+  te::LoopNest loop({output});
+  loop.simplify();
+  loop.prepareForCodegen();
+  te::LLVMCodeGen cg(loop.root_stmt(), {input, weight, bias, output});
```

Comment:
This might need `ifdef TORCH_ENABLE_LLVM`.

Reply:
Is there any other reason for it except the slowness of the IR evaluator?

Reply:
It's probably the reason of the linker errors on this PR: if pytorch is built without LLVM, `te::LLVMCodeGen` will be an undefined symbol.